itou/templates/utils/toasts.html

```diff
@@ -0,0 +1,15 @@
+{% load theme_inclusion %}
+{% for message in messages %}
+    {% if "toast" in message.tags %}
+        <div class="{{ message|itou_toast_classes }}" role="alert" aria-live="assertive" aria-atomic="true" data-delay="3000">
```

Comment:
3 secondes est vraiment court, notamment pour les utilisateurs ayant des besoins en terme d’accessibilité. Aussi, comme on ne peut pas mettre en suspens le timer, l’utilisateur n’a pas nécessairement le temps de lire le contenu du message. Je proposerais bien de retirer l’auto-hide, ou bien de mettre une durée beaucoup plus longue, genre 20 secondes.
```suggestion
        <div class="{{ message|itou_toast_classes }}" role="alert" aria-live="assertive" aria-atomic="true" data-delay="20000">
```